### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by fregante**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -61,13 +61,13 @@
 		"shorten-repo-url": "^2.0.0",
 		"strip-indent": "^3.0.0",
 		"text-field-edit": "^3.1.0",
-		"tiny-version-compare": "2.0.0",
+		"tiny-version-compare": "^3.0.1",
```
Comment (fregante) [inline package.json:64]: I tested this and the difference is minimal

File: source/options.html [hunk 2]
```diff
@@ -1,7 +1,6 @@
 <!doctype html>
 <meta charset="utf-8">
 <title>Refined GitHub options</title>
-<link rel="stylesheet" href="chrome://global/skin/in-content/common.css">
```
Comment (fregante) [inline source/options.html:4]: https://github.com/fregante/webext-base-css/pull/5

File: source/github-helpers/index.ts [hunk 3]
```diff
@@ -2,7 +2,7 @@ import select from 'select-dom';
 import onetime from 'onetime';
 import elementReady from 'element-ready';
 import compareVersions from 'tiny-version-compare';
-import * as pageDetect from 'github-url-detection/esm/index.js'; // eslint-disable-line import/extensions -- Required for Node tests compatibility
+import * as pageDetect from 'github-url-detection';
```
Comment (fregante) [inline source/github-helpers/index.ts:5]: https://github.com/fregante/github-url-detection/pull/37 🎉 hoping this isn't a breaking release for others

File: package.json [hunk 4]
```diff
@@ -42,14 +42,14 @@
 		"debounce-fn": "^4.0.0",
 		"delay": "^4.4.0",
 		"delegate-it": "^2.0.0",
-		"dom-chef": "^4.2.0",
+		"dom-chef": "^4.2.2",
```
Comment (yakov116) [inline package.json:45]: If I remember correctly we have a todo with this one. (Something about in-line css)

Comment (kidonng) [reply package.json:45]: Solved? vadimdemedes/dom-chef#70

Comment (yakov116) [reply package.json:45]: https://github.com/sindresorhus/refined-github/blob/d8c1c195cfdff5031fda6dc688ba36828e9d7d49/source/features/remove-label-faster.tsx#L37

File: source/features/remove-label-faster.tsx [hunk 5]
```diff
@@ -34,32 +34,26 @@ async function removeLabelButtonClickHandler(event: delegate.Event<MouseEvent, H
...
-			label.append(makeRemoveLabelButton(label.dataset.name!, label.style.backgroundColor));
+			label.append(
+				<button
+					type="button"
+					aria-label="Remove this label"
+					className="btn-link tooltipped tooltipped-nw rgh-remove-label-faster"
+					data-name={label.dataset.name}
+					style={/* eslint-disable-line @typescript-eslint/consistent-type-assertions */{
+						'--rgh-remove-label-faster-color': label.style.backgroundColor
+					} as React.CSSProperties}
```
Comment (fregante) [inline source/features/remove-label-faster.tsx:52]: Due to https://github.com/typescript-eslint/typescript-eslint/issues/2587

**Pull request merged**